src/pages/workspaces/index.tsx

```diff
@@ -0,0 +1,63 @@
+import Page from "core/components/Page";
+import { createGetServerSideProps } from "core/helpers/page";
+import useFeature from "identity/hooks/useFeature";
+import useMe from "identity/hooks/useMe";
+import { useRouter } from "next/router";
+import { ReactElement } from "react";
+import { useTranslation } from "react-i18next";
+import CreateWorkspaceDialog from "workspaces/features/CreateWorkspaceDialog";
+import {
+  WorkspacesPageDocument,
+  WorkspacesPageQuery,
+} from "workspaces/graphql/queries.generated";
+
+const WorkspacesHome = () => {
+  const router = useRouter();
+  const [hasWorkspacesEnabled] = useFeature("workspaces");
+  const me = useMe();
+  const { t } = useTranslation();
+
+  const handleClose = (workspaceId: string | null) => {
+    if (!workspaceId) {
```

Comment:
I think you should just do nothing here. Leave the dialog open while we implement a page